src/samples/texturearray/TextureArrayExample.cs

```diff
@@ -192,7 +192,7 @@ void loadTextureArray(string filename, VkFormat format)
             imageCreateInfo.usage = VK_IMAGE_USAGE_SAMPLED_BIT;
             imageCreateInfo.sharingMode = VK_SHARING_MODE_EXCLUSIVE;
             imageCreateInfo.initialLayout = VK_IMAGE_LAYOUT_UNDEFINED;
-            imageCreateInfo.extent = { textureArray.width, textureArray.height, 1 };
+            imageCreateInfo.extent = new  { textureArray.width, textureArray.height, 1 };
```

Comment:
By the way, I never actually finished this sample project. So don't worry if it fails to compile.